test/e2e/e2e_test.go

```diff
@@ -52,7 +52,7 @@ var _ = ginkgo.Describe("Kueue", func() {
 		gomega.Expect(k8sClient.Create(ctx, sampleJob)).Should(gomega.Succeed())
```

Comment:
not related to this PR, but the queue name above can be added like this `testing.MakeJob(name, ns).Queue("main")`

Reply:
Thanks for that!